validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -159,6 +159,8 @@ def can_set(self, address_value_list):
         for add_value_dict in address_value_list:
             for address in add_value_dict.keys():
                 if address not in self._write_list:
+                    LOGGER.warning("Authorization exception, address: %s",
```

Comment:
We should create a story that actually sends back the error that occurred in the txn processor.